nginx/conf/nginx.conf

```diff
@@ -99,6 +99,7 @@ http {
 	# Virtual Host Configs
 	##
 
-	include /etc/nginx/conf.d/*.conf;
```

Comment:
@kirtangajjar Where is the mime.types file getting included?

Reply:
Also, after you make the changes, please test and ensure the container is running and it is loading all the files in sequence.

You can run `nginx -T` to check the whole configuration file loaded.